README.rdoc

```diff
@@ -677,6 +677,7 @@ minitest-unordered          :: Adds a new assertion to minitest for checking the
                                contents of a collection, ignoring element order.
 minitest-vcr                :: Automatic cassette managment with Minitest::Spec
                                and VCR.
+minitest_log                :: Adds structured logging, data explication, and verdicts.
```

Comment:
Anything more needed here?
* Gem: https://rubygems.org/gems/minitest_log
* Code: https://github.com/BurdetteLamar/minitest_log

Reply:
If not to be merged, should I close?

Reply:
cool your jets.